enterprise/server/vmexec/vmexec_test.go

```diff
@@ -140,7 +140,7 @@ func TestExecStreamed_Crash(t *testing.T) {
 		Arguments: []string{"bash", "-c", `
 			echo foo-stdout >&1
 			echo bar-stderr >&2
-			kill 0 -KILL
```

Comment:
any idea why this change is needed? I'd be very surprised if upgrading to rules_go broke this

Reply:
https://buildbuddy.buildbuddy.io/invocation/d1ddf6d1-cf88-4de1-89bd-bf39c97ca43d here is an example of this failure.

From https://man7.org/linux/man-pages/man1/kill.1.html, we should use `kill -KILL -- 0` to be sure.

It's unclear to me why https://github.com/bazelbuild/rules_go/pull/3920 triggered this though.